server/src/test/java/org/cloudfoundry/identity/uaa/audit/JdbcAuditServiceTests.java

```diff
@@ -41,7 +41,7 @@ public void userAuthenticationFailureAuditSucceeds() throws Exception {
         auditService.log(getAuditEvent(UserAuthenticationFailure, "1", "joe"));
         Thread.sleep(100);
         auditService.log(getAuditEvent(UserAuthenticationFailure, "1", "joe"));
-        List<AuditEvent> events = auditService.find("1", 0);
+        List<AuditEvent> events = auditService.findInZone("1", IdentityZone.getUaa().getId(), 0);
```

Comment:
This change is no needed. And is useful to ensure that the system works correctly with the method override.